external-crates/move/crates/move-package-alt/src/errors/mod.rs

```diff
@@ -110,6 +110,11 @@ pub enum PackageError {
 
     #[error("Multiple entries with `source = {{ {dep} }}` exist in the publication file")]
     MultipleEphemeralEntries { dep: String },
+
+    #[error(
+        "Cannot override default environments. Environment `{name}` is a system environment and cannot be overridden."
```

Comment:
We could consider listing the default environments here, but I don't think it's necessary